policy/modules/contrib/kdump.te

```diff
@@ -63,6 +63,7 @@ files_read_kernel_modules(kdump_t)
 files_read_kernel_img(kdump_t)
 files_map_boot_files(kdump_t)
 
+kernel_load_unsigned_module(kdump_t)
```

Comment:
I looked up the RHEL-9 BZ behind this and the operation that kdump is doing is not a kernel module load, but rather `kexec_load(2)`, which allows to specify a kernel image to reboot into (usually this is used to do analysis/memory dump after a system crash). This currently doesn't support signature verification of the kernel being loaded, so it always asks for lockdown integrity permission.

I'd prefer to create a new interface for this (e.g. `kernel_kexec_load()`), which would grant both the `sys_boot` capability ([also required unconditionally for this syscall](https://elixir.bootlin.com/linux/v5.11/source/kernel/kexec.c#L204)) and the `lockdown` `integrity` permission.